bumpversion/__init__.py

```diff
@@ -72,7 +81,7 @@ def commit(cls, message):
         f.write(message.encode('utf-8'))
         f.close()
         env = os.environ.copy()
-        env['HGENCODING'] = 'utf-8'
+        env[str('HGENCODING')] = str('utf-8')
         try:
             subprocess.check_output(cls._COMMIT_COMMAND + [f.name], env=env)
```

Comment:
Odd that this dosen't need encoding.

Reply:
f is a temporary file, and its name is ascii